tests/conftest.py

```diff
@@ -1,22 +1,19 @@
+import imp
 import logging
-import time
-import pytest
-from common.switch import Sai
 import os
-import sys
-import imp
-import signal
 import random
+import shutil
+import signal
+import sys
+import time
 import unittest
 
-curdir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
redundant, as ptf is a installed python module. ptf can be directly imported from python modules library.